packages/nuxt/src/module.ts

```diff
@@ -208,7 +208,9 @@ export default defineNuxtModule<VueFireNuxtModuleOptions>({
       }
 
       // loads the user on the current app
-      addPlugin(resolve(runtimeDir, 'auth/plugin-authenticate-user.server'))
+      if(!options.auth.clientOnly){
```

Comment:
```suggestion
      if (!options.auth.clientOnly) {
```
Is the key always set on auth?

Reply:
The clientOnly key?
If the key is not set it executes the previously default behaviour of loading the auth/plugin-authenticate-user.server.